CHANGELOG.rst

```diff
@@ -32,6 +32,7 @@ Notable improvements and fixes
   for empty commands.
 - The ``test`` builtin now better shows where an error occured (#6030).
 - Add a helper function to know if the user is root (#7031).
+- `fish_postexec` has a second argument that's set to `1` when the `$status` is a holdover from the previous command (e.g. because the current command was backgrounded, or was a variable assignment), `0` otherwise.
```

Comment:
Unfortunately, since we switched this to `rst` from `md` we need two backticks instead of one.

Reply:
Done